### DISCUSSION TITLE
manual bed leveling sizing

### DISCUSSION THREAD

**Discussion by Failure2Society** [Jyers/Marlin#473] [Q&A] [answered]
I have modified my Ender 3 V2 to have a 400mm x 400mm bead, and the manual bed level only does the points (32.5,32.5),(32.5,197.5),(197.5,197.5),(197.5,32.5) square, and I have messed around with my own firmware and have not been able to adjust the sizing from that. do you have an idea on what to do to adjust this?

**Comment by aryajuanda** [comment]
you can start here :

https://github.com/Jyers/Marlin/blob/b61f92ab58f6477e297c31201ca90429d6dcf1e7/Marlin/src/lcd/extui/lib/creality_dwin/creality_dwin.cpp#L1159

**Reply by Failure2Society** [reply]
thank you, that seems to fix the issue

**Comment by Jyers** [comment, marked as answer]
I'm going to make this more universal soon so the firmware can account for larger beds.
